### PR TITLE
WP-497 spec ldap config

### PR DESCRIPTION
**Pull request by bloom1**
Do not merge, WAZO-2653 https://github.com/wazo-platform/wazo-auth/pull/200 is based on this PR and this will be merged with it :smile:

### CODE REVIEW

File: wazo_auth/plugins/http/ldap_config/api.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+paths:
+  /backends/ldap_user:
```
Comment (fblackburn1) [inline wazo_auth/plugins/http/ldap_config/api.yml:2]: Preference for `/backends/ldap`

File: wazo_auth/plugins/http/ldap_config/api.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      tags:
+        - backends
+      parameters:
+        - name: body
+          in: body
+          description: The LDAP backend configuration
+          required: true
+          schema:
+            $ref: '#/definitions/LDAPBackendConfig'
+        - $ref: '#/parameters/tenantuuid'
```
Comment (fblackburn1) [inline wazo_auth/plugins/http/ldap_config/api.yml:47]: Should be before the body to follow other API style

File: wazo_auth/plugins/http/ldap_config/api.yml (hunk 1 above)
Comment (bloom1) [reply wazo_auth/plugins/http/ldap_config/api.yml]: Nice, indeed it is better. Thank you :smiley:

File: wazo_auth/plugins/http/ldap_config/api.yml (hunk 2 above)
Comment (bloom1) [reply wazo_auth/plugins/http/ldap_config/api.yml]: I agree, thank you :smile:

**Review by fblackburn1 — APPROVED** [review]

**Comment by bloom1** [comment]
Merged in PR #200

**Pull request closed without merging**